core/coreapi/unixfs_test.go

```diff
@@ -0,0 +1,25 @@
+package coreapi
+
+import (
+	"github.com/magiconair/properties/assert"
```

Comment:
Could you switch to https://github.com/stretchr/testify? It is more widely used by golang community